services/id-allocation-service/src/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Hono } from 'hono'
+import type { DurableObjectNamespace } from '@cloudflare/workers-types'
+import { DurableObject } from 'cloudflare:workers'
+
+export type Bindings = {
+  ID_ALLOCATION: DurableObjectNamespace<IdAllocation>
+}
+
+const app = new Hono<{ Bindings: Bindings }>()
+
+type Ids = { [key: number]: number }
+
+export class IdAllocation extends DurableObject {
+  async allocateId(): Promise<number> {
+    const now = Date.now()
+    const ids: Ids = JSON.parse((await this.ctx.storage.get('ids')) || '{}')
+    for (let id = 0; ; id++) {
+      const exp = ids[id]
+      const isFree = exp === undefined || exp < now
+      if (isFree) {
+        ids[id] = now + 1000 * 60 * 5
```

Comment:
Shouldn't this match the 18_000 from the test timeouts + a little buffer? 

I think this miiiight cause some issues?

Reply:
The tests have like 3 minutes of timeouts

This sets it to 5 minutes, so that's plenty of buffer